build-ts/lib/client.js

```diff
@@ -254,7 +270,7 @@ export default class CommonClient extends EventEmitter {
                 if (message.method) {
                     // run as microtask so that pending queue messages are resolved first
                     return Promise.resolve().then(() => {
-                        this.emit(message.method, message === null || message === void 0 ? void 0 : message.params);
```

Comment:
This is one of those classic examples of how building for a higher target can yield bundle size savings for a consumer that doesn't need es2017 compatibility.